### ISSUE TITLE
Feature: Get list of blocked account ids

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#741]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.0.2](https://github.com/open-wa/wa-automate-nodejs/compare/2.0.1...2.0.2) (2020-08-24)

- Release 2.0.2 [`ca3e0c0`](https://github.com/open-wa/wa-automate-nodejs/commit/ca3e0c0fe41e3a265667f8a9b720aa6c51e2948d)
- added `tagEveryone` to docs [`5465b03`](https://github.com/open-wa/wa-automate-nodejs/commit/5465b03ad34c47ba78468a9a1e111dd469c5d369)
- Feature: Implemented `getBlockedIds` #741 [`20ef72c`](https://github.com/open-wa/wa-automate-nodejs/commit/20ef72c26bf01502146475f96c763fd0f39c192d)
- improved license documentation [`5f83183`](https://github.com/open-wa/wa-automate-nodejs/commit/5f8318397ef545fad6029d94cc8cd989c00c592f)
- executable can now consume base64 encoded session data with `-s` flag [`6db1449`](https://github.com/open-wa/wa-automate-nodejs/commit/6db14494e8f88277fc064ab94fc4005da66365f8)
